build.sbt

```diff
@@ -25,14 +25,14 @@ test in assembly := {}
 
 assemblyShadeRules in assembly := {
   val shadePackage = "geotrellis.sdg.shaded"
-  val gtVersion = "2.0.0"
+  val gtVersion = "2.1.0"
```

Comment:
I also bumped this to be what osmesa expects: https://github.com/azavea/osmesa/blob/master/src/project/Version.scala#L4

I _think_ the resolution to this issue was to both swap the gt version to the right one, and to ensure we're shading the actual dependency where the WKT class gets defined, which was geotrellis-vector. I've had some trouble verifying though that compilation in databricks fails consistently without both changes. Perhaps we can work through this before merge.